ecommerce-ios/Views/ProductInformation/ProductInformationView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ProductInformationView.swift
+//  ecommerce-ios
+//
+//  Created by Nguyen M. Tam on 19/05/2021.
+//
+
+import SwiftUI
+
+struct ProductInformationView: View {
+
+    let viewModel: ProductInformationViewModel
+
+    var body: some View {
+        VStack {
+            Image(viewModel.imageString)
+                .resizable()
+                .scaledToFit()
+                .padding(.vertical, 16.0)
+
+            VStack(spacing: 4.0) {
+                Text(viewModel.productName.capitalized)
+                    .font(.system(size: 28.0).weight(.bold))
+                    .foregroundColor(.charadeGray)
+                Text("\(viewModel.price.currency)\(viewModel.price.amount)")
+                    .font(.system(size: 22.0))
+                    .foregroundColor(.charadeGray)
+            }
+            .padding(.vertical, 16.0)
+
+            Button(
+                action: { print("did tap add to cart button") },
+                label: {
+                    Text("Add to cart")
+                        .font(.system(size: 17.0).weight(.semibold))
+                        .foregroundColor(.white)
+                }
+            )
+            .frame(width: screenWidth - 32.0, height: 50.0)
```

Comment:
what do you think about
```

            .frame(maxWidth: .infinity, minHeight: 50.0)
            .padding(.horizontal, 32.0)
```
so we can make these view more responsive.

Reply:
It's great to know it.